requirements.in

```diff
@@ -33,6 +33,7 @@ SQLAlchemy
 statsd
 stripe
 timestring
+urllib3>=1.26.18
```

Comment:
i think this is fine, but because you updated this in `shared` in https://github.com/codecov/shared/pull/70 you could avoid adding urllib3 to requirements.in by updating shared if you wanted to